examples/python/KinDynComputationsTutorial.py

```diff
@@ -40,19 +41,28 @@
 dynComp.setRobotState(s,ds,gravity);
 
 jac = iDynTree.MatrixDynSize(6,6+dofs);
-ok = dynComp.getFreeFloatingJacobian("lf_foot", jac);
+ok = dynComp.getFrameFreeFloatingJacobian("l_sole", jac);
 if( not ok ):
-    print "Error in computing jacobian of frame " + "lf_foot";
-else: 
-    print "Jacobian of lf_foot is\n" + jac.toString();
+    print("Error in computing jacobian of frame " + "l_sole");
+else:
+    print("Jacobian of lf_foot is\n" + jac.toString());
 
 
+root_H_sole = dynComp.getRelativeTransform("root_link", "l_sole");
+root_R_sole = root_H_sole.getRotation()
+if( not ok ):
+    print("Error in computing jacobian of frame " + "l_sole");
+else:
+    print("Orientation between root_link and l_sole is\n" + str(root_R_sole.toNumPy()));
+    print("In RPY format:\n" + str(root_R_sole.asRPY().toNumPy()))
+
 baseAcc = iDynTree.Vector6();
 baseAcc.zero();
 links = dynComp.model().getNrOfLinks();
 regr = iDynTree.MatrixDynSize(6+dofs,10*links);
 ok = dynComp.inverseDynamicsInertialParametersRegressor(baseAcc, dds, regr);
-if( not ok ):
-    print "Error in computing the dynamics regressor";
-else :
-    print "The dynamics regressor is\n" + regr.toString();
+# The Dynamics Regressior is quite big, uncomment if you really want to print it
+#if( not ok ):
+#    print("Error in computing the dynamics regressor");
+#else :
+#    print("The dynamics regressor is\n" + regr.toString());
```

Comment:
Is there a reason why this part is commented?

Reply:
It is written in the first part of the comment, " The Dynamics Regressior is quite big, uncomment if you really want to print it". We can also remove it, but the identification of inertial parameters was actually the first thing I ever worked during my master thesis, so I like to include it in the tutorial. :D